assets/styles/index.scss

```diff
@@ -1,3 +1,8 @@
+// The $govuk-font-family var must be set in order to prevent govuk-frontend components injecting their own 'nta'
+// font-face declarations. If you choose to bring components directly into your project from govuk-frontend rather than
+// via the proxies provided here, then you must make sure this var is declared before those includes.
```

Comment:
This lot should go in the README ~instead of~ as well as being hidden in code.

On that note, I think the README needs a full makeover now.